x-pack/platform/plugins/shared/onechat/server/services/runner/types.ts

```diff
@@ -22,6 +23,7 @@ export interface RunnerFactoryDeps {
   security: SecurityServiceStart;
   // plugin deps
   inference: InferenceServerStart;
+  spaces: SpacesPluginStart | undefined;
```

Comment:
Not a type mistake - `SpacesPluginStart | undefined` makes sure the dependency is passed down (but the spaces plugin can be disabled, in which case the dep is `undefined`)